Makefile

```diff
@@ -51,3 +51,7 @@ push:
 .PHONY: generate
 generate:
 	go generate ./pkg/...
+
+.PHONY: lint
+lint:
+	./hack/lint.sh
```

Comment:
Why using lint.sh file and not running the go command from here?

Reply:
Is easier to do complex bas a bash script instead of bash snippet at Makefiles, and it's also a common pattern around, the Makefile is the entrypoint for the project and you have helper scripts.

Reply:
Yes, I agree that it easier to handle complex scripting this way
But this is not the case :-) its only one liner
I think calling it from the Makefile is better this case

Reply:
Moved to Makefile and parameterized golangci-lint version